### PR TITLE
Drop outdated code

### PR DESCRIPTION
**Pull request by fregante**

### CODE REVIEW

**Review by fregante — COMMENTED** [review]
testing

File: source/features/one-click-review-submission.tsx [hunk 1]
```diff
@@ -3,16 +3,15 @@ import delegate, {DelegateEvent} from 'delegate-it';
...
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 import {assertNodeContent} from '../helpers/dom-utils.js';
 
 function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 	const form = originalSubmitButton.form!;
-	const actionsRow = originalSubmitButton.closest('.Overlay-footer');
-	// TODO: For GHE. Remove after June 2024
-	const legacyActionsRow = originalSubmitButton.closest('.form-actions')!;
+	const actionsRow = originalSubmitButton.closest('.Overlay-footer')!;
```
Comment (busches) [inline source/features/one-click-review-submission.tsx:14]: Looks like dropping this has broken GHE 3.12.3, I get `actionRow is null` now. Where'd we get the comment from?

File: source/features/one-click-review-submission.tsx [hunk 2]
```diff
@@ -34,22 +33,14 @@ function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 		);
 	}
 
-	if (actionsRow) {
```
Comment (busches) [inline source/features/one-click-review-submission.tsx:37]: My comment should have been here 😞

File: source/features/one-click-review-submission.tsx (hunk 1 above)
Comment (fregante) [reply source/features/one-click-review-submission.tsx:14]: Feel free to revert changes to this file with an updated 2025 comment. It's weird that GitHub hasn't updated the DOM at all

Comment (yakov116) [reply source/features/one-click-review-submission.tsx:14]: Please open a PR

**Pull request merged**